<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>600</y>
      <width>122</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setRateButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>630</y>
      <width>122</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>rate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rateEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>630</y>
      <width>113</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setWeekButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>630</y>
      <width>122</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>week</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>50</y>
      <width>281</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QLabel" name="rateLabel">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>90</y>
      <width>281</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="numberOfAgentsLabel">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>130</y>
      <width>271</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string># of agents</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="weekEdit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>630</y>
      <width>113</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dayOfWeekEdit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>690</y>
      <width>113</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="setDayOfWeekButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>690</y>
      <width>122</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>day of week</string>
    </property>
   </widget>
   <widget class="QPushButton" name="timeButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>690</y>
      <width>122</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>690</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="banBridgeButton1">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>670</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>only bridge1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="banBridgeButton2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>670</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>only bridge2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1077</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
